OpenJDK contains very outdated autoconf-config.guess and autoconf-config.sub
that don't handle triplet with -musl. It also contains wrapper scripts
config.guess and config.sub that we don't need at all.

This patch modifies these wrapper scripts to just include up-to-date
config.guess and config.sub from the icedtea tarball.

--- openjdk.orig/common/autoconf/build-aux/config.guess
+++ openjdk/common/autoconf/build-aux/config.guess
@@ -28,62 +28,4 @@
 # and fix the broken property, if needed.
 
 DIR=`dirname $0`
-OUT=`. $DIR/autoconf-config.guess`
-
-# Test and fix solaris on x86_64
-echo $OUT | grep i386-pc-solaris > /dev/null 2> /dev/null
-if test $? = 0; then
-  # isainfo -n returns either i386 or amd64
-  REAL_CPU=`isainfo -n`
-  OUT=$REAL_CPU`echo $OUT | sed -e 's/[^-]*//'`
-fi
-
-# Test and fix solaris on sparcv9
-echo $OUT | grep sparc-sun-solaris > /dev/null 2> /dev/null
-if test $? = 0; then
-  # isainfo -n returns either sparc or sparcv9
-  REAL_CPU=`isainfo -n`
-  OUT=$REAL_CPU`echo $OUT | sed -e 's/[^-]*//'`
-fi
-
-# Test and fix cygwin on x86_64
-echo $OUT | grep 86-pc-cygwin > /dev/null 2> /dev/null
-if test $? != 0; then
-  echo $OUT | grep 86-pc-mingw > /dev/null 2> /dev/null
-fi
-if test $? = 0; then
-  case `echo $PROCESSOR_IDENTIFIER | cut -f1 -d' '` in
-    intel64|Intel64|INTEL64|em64t|EM64T|amd64|AMD64|8664|x86_64)
-      REAL_CPU=x86_64
-      OUT=$REAL_CPU`echo $OUT | sed -e 's/[^-]*//'`
-      ;;
-  esac
-fi
-
-# Test and fix architecture string on AIX
-# On AIX 'config.guess' returns 'powerpc' as architecture but 'powerpc' is
-# implicitely handled as 32-bit architecture in 'platform.m4' so we check
-# for the kernel mode rewrite it to 'powerpc64' if we'Re running in 64-bit mode.
-# The check could also be done with `/usr/sbin/prtconf | grep "Kernel Type" | grep "64-bit"`
-echo $OUT | grep powerpc-ibm-aix > /dev/null 2> /dev/null
-if test $? = 0; then
-  if [ -x /bin/getconf ] ; then
-    KERNEL_BITMODE=`getconf KERNEL_BITMODE`
-    if  [ "$KERNEL_BITMODE" = "32" ]; then
-      KERNEL_BITMODE=""
-    fi
-  fi
-  OUT=powerpc$KERNEL_BITMODE`echo $OUT | sed -e 's/[^-]*//'`
-fi
-
-# Test and fix little endian PowerPC64.
-# TODO: should be handled by autoconf-config.guess. 
-if [ "x$OUT" = x ]; then
-  if [ `uname -m` = ppc64le ]; then
-    if [ `uname -s` = Linux ]; then
-      OUT=powerpc64le-unknown-linux-gnu
-    fi
-  fi
-fi
-
-echo $OUT
+. $DIR/../../../../config.guess
--- openjdk.orig/common/autoconf/build-aux/config.sub
+++ openjdk/common/autoconf/build-aux/config.sub
@@ -28,37 +28,4 @@
 # wrap it and fix the broken property, if needed.
 
 DIR=`dirname $0`
-
-# First, filter out everything that doesn't begin with "aarch64-"
-if ! echo $* | grep '^aarch64-' >/dev/null ; then
-    . $DIR/autoconf-config.sub "$@"
-    # autoconf-config.sub exits, so we never reach here, but just in
-    # case we do:
-    exit
-fi
-
-while test $# -gt 0 ; do
-    case $1 in 
-        -- )   # Stop option processing
-            shift; break ;;
-        aarch64-* )
-            config=`echo $1 | sed 's/^aarch64-/arm-/'`
-            sub_args="$sub_args $config"
-            shift; ;;
-        - )    # Use stdin as input.
-            sub_args="$sub_args $1"
-            shift; break ;;
-        * )
-            sub_args="$sub_args $1"
-            shift; ;;
-    esac
-done
-
-result=`. $DIR/autoconf-config.sub $sub_args "$@"`
-exitcode=$?
-
-result=`echo $result | sed "s/^arm-/aarch64-/"`
-
-echo $result
-exit $exitcode
-
+. $DIR/../../../../config.sub "$@"
